### DISCUSSION TITLE
Klaviyo and other plugins available?

### DISCUSSION THREAD

**Discussion by jhaidar21** [saleor/saleor#9701] [Deployment]
On the Saleor site under Integrations I see that Klaviyo and Shipstation are supported but I don't see them in the integration list after deploying Saleor (configuration > plugins). Are there packages that needs to be installed to get these services?

**Comment by karolkielecki** [comment]
Hi @jhaidar21, we're working on them. Not sure about ETA. They won't be available as plugins--they're being developed as apps. And they will be available in the marketplace.
